tests/integration/secret_outputs/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "typescript",
+    "devDependencies": {
+        "@types/node": "latest"
+    },
+    "dependencies": {
+        "@pulumi/pulumi": "latest",
+        "@pulumi/random": "^0.5.4"
```

Comment:
ideally use "dev" for pulumi/random.

Reply:
Will actually remove both of these.  The Random usage was from before I moved to use dynamic providers to remove a dependency.